example.config.toml

```diff
@@ -1,6 +1,11 @@
+# All fields are required for now
 [general]
+token_size = 123
 # Telegram ID of the master account
 masterid = 1234578
 
 [database]
+host = "0.0.0.0"
```

Comment:
I think `127.0.0.1` makes more sense as a default

Reply:
I guess 0.0.0.0 is useful when working with dockers. Not sure though.

Will revert it back anyways! :+1: